working/manual_checks_after_generic.sql

```diff
@@ -595,8 +595,8 @@ ORDER BY LEAST (a.valid_start_date, b.valid_start_date) DESC,
 
 -- 02.13. Mapping of visit concepts
 --In this check we manually review the mapping of visits to the 'Visit' domain.
---The first script is highly sensitive and adjusted for the Procedure vocabularies only.
--- For other vocabularies (Conditions, Measurements, Drug) please use the second script provided below. It's less sensitive and more specific.--To prioritize and make the review process more structured, the logical groups to be identified using the sorting by flag, flag_visit_should_be and vocabulary_id fields. Then the content to be reviewed separately within the groups.
+--The first script (02.13.01) is highly sensitive and adjusted for the Procedure vocabularies only.
```

Comment:
@m-khitrun Would not work like that. Let's keep the descriptions together with the respective scripts

Reply:
Fixed.